jp/components/calendar.md

```diff
@@ -286,11 +286,12 @@ This is the result.
 </div>
 
 
-### Special dates
+### 特定の日付
 
-`Special dates` feature is using almost the same configuration principles as `Disabled dates`. The difference here is dates `styling` and `interaction`. You are able to select and focus `Special date`.
+`Special dates` 機能は、`Disabled dates` とほとんど同じ構成を使用します。違いは、日付の `styling` と `interaction` です。
 
-Let's add a `Special dates` to our `igxCalendar`, we are going to create a `DateRangeDescriptor` item of type `DateRangeType.Specific` and pass array of dates as `dateRange`:
+また `Special date` の選択やフォーカスが可能です。
```

Comment:
This sentence should be in the previous paragraph.  When there is a blank line between sentences, they will be in a new paragraph.